src/renderer/webgl/WebglRenderer.ts

```diff
@@ -95,22 +193,59 @@ export class WebglRenderer extends EventEmitter implements IRenderer {
     // Detect whether IntersectionObserver is detected and enable renderer pause
     // and resume based on terminal visibility if so
     if ('IntersectionObserver' in window) {
-      const observer = new IntersectionObserver(e => this.onIntersectionChange(e[0]), { threshold: 0 });
+      const observer = new IntersectionObserver(e => this._onIntersectionChange(e[0]), { threshold: 0 });
       observer.observe(this._terminal.element);
       this.register({ dispose: () => observer.disconnect() });
     }
+
+    WebglRenderer._contextHelper.onRendererCreated(this);
   }
 
   public dispose(): void {
     this._renderLayers.forEach(l => l.dispose());
     this._terminal.screenElement.removeChild(this._canvas);
     super.dispose();
+    this._canvas.width = 0;
+    this._canvas.height = 0;
+
+    if (!this._gl.isContextLost()) {
+      console.log('Force context loss');
```

Comment:
This should happen automatically after dispose on the next GC?

Reply:
Yes, however, we force it so that another context can be created before the next GC. If we don't force it here, Chrome might not notice there's an available context and could kill an active one.

What I observed is:
  1. open 20 tabs (the first 4 will lose their context)
  1. close the last 16 tabs
  1. close one more tab
  1. **result**: Chrome will warn that the max number of contexts has been reached
  1. **expected**: no warning

So, it seems like one of these is happening:
  1. GC is not firing during the closing of the 17 tabs
  1. GC doesn't really collect webgl contexts
  1. Someone's keeping a ref to this instance, preventing it from being GC

I'll grab a snapshot to make sure (3) is not happening